code/modules/projectiles/projectile/special.dm

```diff
@@ -183,12 +183,12 @@
 	name ="armor-piercing shell"
 	icon = 'icons/obj/grenade.dmi'
 	icon_state = "shell"
-	damage = 90
+	damage = 60

```

Comment:
i think 90 damage as a direct hit would've been okay anyway, in the grand scheme of someone probably getting their face turned into a bursted watermelon by an AP shell.